### ISSUE TITLE
/party command to help groups/bands of players communicate easier in-game (suggestion not issue)

### ISSUE THREAD

**Issue by LabtechsOnGitHub** [RemainingToast/2b2tau-issues#5]
a /party command would help because when you are getting multiple dms you have to go through the hassle of /r and /msg, so with /party chat u dont need to type any command and can communicate easily.

**Comment by RhysyPiece360** [comment]
Would be even more useful for groups doing stuff

**Comment by LabtechsOnGitHub** [comment]
yeah actually, i wanted to use it today when i was exploring with my team

**Comment by RhysyPiece360** [comment]
I’m hank01 btw

**Comment by RhysyPiece360** [comment]
My pc is packed up rn
